README.md

```diff
@@ -1,24 +1,134 @@
-# README
+<a name="readme-top"></a>
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Hello @HAMMAS-SALEEM,
Thank you so much for the feedback. I will make sure to have a logo as soon as possible.
Cheers.